.gitlab-ci-darwin.yml

```
@@ -0,0 +1,95 @@
+variables:
+  SCHEDULER_PARAMETERS: '--nodes=1 --partition=power9 --export=NONE'
+  GIT_SUBMODULE_STRATEGY: recursive
+  MODULE_CMAKE: "cmake/3.12.4"
+  MODULE_CLANG: "clang/8.0.1"
+  MODULE_COMPILER: "gcc/7.4.0"
+  MODULE_CUDA: "cuda/10.1"
+  MODULE_MPI: "openmpi/p9/4.0.2-gcc_7.4.0"
+  NUM_GPU_DEVICES_PER_NODE: 2
+  NUM_MPI_PROC_TESTING: 2
+  OMP_NUM_THREADS: 40
```

```suggestion
  OMP_NUM_THREADS: 1
```

As long as we don't have proper threading support I strongly suggest to set num threads to 1. On the machine I using using more threads significantly slows down the code (in addition to technically not being properly supported).